apps/web/src/server/applications/case/s51/applications-s51.mapper.js

```diff
@@ -0,0 +1,131 @@
+/** @typedef {import('../../applications.types.js').S51Advice} S51Advice */
```

Comment:
already existing code: I just moved it here so that the controller is not too noisy